deploy/operator.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: astarte-operator
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: astarte-operator
+  template:
+    metadata:
+      labels:
+        name: astarte-operator
+    spec:
+      serviceAccountName: astarte-operator
+      containers:
+        - name: astarte-operator
+          # Replace this with the built image name
+          image: astarte/astarte-kubernetes-operator:0.10-snapshot
```

Comment:
is this supposed to be hardcoded?
If yes, we should make sure to have some README that helps people in finding hardcoded values that should be changed.

Reply:
It is. This is the deployment of the operator itself. Arguably, we want to have snapshot in master and 0.10-snapshot in release-0.10

Reply:
We must have some sort of checklist with all the required changes that we should do before releasing stuff. Furthermore I think we should change this every time we are going to tag a release, so some sort of README would be nice.